src/components/Router/StepsRouter.js

```diff
@@ -7,6 +7,17 @@ import theme from '../Theme/style.css'
 import { withFullScreenState } from '../FullScreen'
 
 class StepsRouter extends Component {
+  componentDidUpdate(prevProps) {
+    // Re-focus to container is needed for accessibility to have a correct
+    // tabbing order, and should be triggered when...
+    if (this.container &&
+        // ...step changes (for tabbing order to start from the top)
+        (prevProps.step !== this.props.step) ||
+        // ..."full screen" mode changes (e.g. for enlarged image preview)
+        (prevProps.isFullScreen !== this.props.isFullScreen)) {
```

Comment:
hm, interesting approach. I wonder if there is a fullscreen switch that might not beg for a refocus.

Reply:
The only place I found where this would not need a switch when entering a camera mode. However, because at that point a step is changed, the focus would still happen due to the condition on the line above.

This is not ideal solution, I agree, but have no other idea for now...